examples/storage_pools.py

```diff
@@ -32,7 +33,7 @@
 
 # Try load config from a file (if there is a config file)
 config = try_load_from_file(config)
-oneview_client = OneViewClient(config)
+oneview_client = OneViewClient.from_json_file('config-rename.json')
```

Comment:
we should rename config_rename.json to config.json. So please change file_name here